src/test/rbd_mirror/image_replayer/test_mock_CreateImageRequest.cc

```diff
@@ -309,12 +320,10 @@ class TestMockImageReplayerCreateImageRequest : public TestMockFixture {
 
 TEST_F(TestMockImageReplayerCreateImageRequest, Create) {
   librbd::MockTestImageCtx mock_remote_image_ctx(*m_remote_image_ctx);
-
-  CreateCloneImage create_clone_image;
+  MockCreateRequest mock_create_request;
 
   InSequence seq;
-  expect_create_image(create_clone_image, "image name", "global uuid",
-                      "remote uuid", 0);
+  expect_create_image(mock_create_request, 0);
```

Comment:
Minor: you can add a MOCK_METHOD(construct, void(<image name>, <global uuid>, <remote mirror uuid>)) and validate the correct values are passed to the constructor.